tutorials/09-variational-inference/Project.toml

```diff
@@ -2,11 +2,12 @@
 Bijectors = "76274a88-744f-5084-9051-94815aaf08c4"
 ComponentArrays = "b0b7db55-cfe3-40fc-9ded-d10e2dbeff66"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
+Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
```

Comment:
I think Flux is not used anywhere?
```suggestion
```
(requires to rebuild Manifest.toml)

Reply:
@devmotion I guess it's used in Line 362: 
```
vi(m, advi; optimizer=Flux.ADAM())
```
as mentioned in Line 356: "With that being said, thanks to Requires.jl, if we add a `using Flux` prior to `using Turing` we can also make use of all the optimizers in `Flux`, e.g. `ADAM`, without any additional changes to your code! For example ..."

Reply:
Ah Isee!